src/lib.rs

```diff
@@ -1839,6 +1840,37 @@ impl OpAttrDef {
     }
 }
 
+/// Types of attribute values for an operation
+#[non_exhaustive]
+#[derive(Debug)]
+pub enum AttrValue {
+    S(Vec<u8>), 
+    I(i64), 
+    F(f32), 
+    B(bool), 
+    Field_type(protos::types::DataType), 
```

Comment:
This should be the native Rust enum, not the protobuf enum (i.e. `tensorflow::DataType`, not `protos::types::DataType`).